README.md

```diff
@@ -2,7 +2,15 @@
 
 A webpack plugin for running single-spa microfrontends in standalone mode. This is an alternative to using [import-map-overrides](https://github.com/joeldenning/import-map-overrides).
 
-_Running in standalone mode is not 100% equivalent to when it's fully integrated into the rest of the app. For this reason, using import-map-overrides on an integrated environment is generally recommended._
+> **⚠️ Warning: Experimental**
+>
+> This plugin should be considered "experimental" in that the core team has not yet settled on a consensus as to if 'standalone mode' is how local development should be approached. Generally speaking, developers want this because "that's the way we used to do it". **There are potential pitfalls that you should understand before deciding to use this plugin.**
+>
+> - 'standalone' also means isolated which may lead to situations of "works on my machine, but not on the deployed environment."
+> - The import map used in this plugin's configuration may quickly become outdated if you use any sort of versioning (which is recommended as a best-practice in general). This may lead to developing locally against a package whose API has changed.
```

Comment:
```suggestion
> - This plugin always uses the latest versions of SystemJS and single-spa, which is likely different than most environments, which version their dependencies.
```

Reply:
These seem to be different and separate points to me. Mine is talking about how a hard-coded import map in this plugin's config can quickly become outdated. Yours talks about automatic upgrades to SystemJS and single-spa script tags. I think they're should both be included.

Reply:
I see, yes that's true. Regarding hard coding the import map - the better solution is to provide the `importMapUrl`, which allows you to always use the latest deployed version of your other microfrontends. See for instance this code:

https://github.com/react-microfrontends/people/blob/25f18f22ee89fea827db02cb92d3e021ea514938/webpack.config.js#L22-L24

If you do it this way, there is no more issue.